recipe/build.sh

```diff
@@ -70,3 +70,15 @@ EOF
         touch $PREFIX/Emacs.app/Contents/MacOS/libexec/Emacs.pdmp
     fi
 fi
+
+if [ "$(uname)" == "Linux" ]; then
+    mkdir -p $PREFIX/share/emacs/site-lisp
+    cat <<EOF > $PREFIX/share/emacs/site-lisp/site-start.el
```

Comment:
Yes, but it's possible that it will be introduced in other parts of the packaging in the future, and adding another `>` seems like a trivial thing that wouldn't hurt much. But as I said, feel free to ignore. :+1:

Reply:
`>>` would do nothing. This is done at the build step. Whatever file is created in the build is what is shipped with the package. If this file is somehow included with other packages and we want to append it, that will have to be done in a post-install script. Although I'm unclear what other packages would ship files in `$PREFIX/share/emacs`.